newIDE/app/src/ResourcesList/LocalPiskelBridge.js

```diff
@@ -52,7 +52,7 @@ export const openPiskel = ({
       outputResources.forEach(resource => {
         const imageResource = new gd.ImageResource();
         imageResource.setFile(path.relative(projectPath, resource.path));
-        imageResource.setName(resource.name);
```

Comment:
Looks good :) Is name needed anymore in Piskel? We can maybe remove it from the object being returned from Piskel if it's dead code?

Reply:
yes indeed. I will remove it

Reply:
@4ian sorry for the oversight. I need to look at that again before merge!
I have a regression in this pull that needs to be fixed with piskel